test/specs/sorting.test.js

```diff
@@ -0,0 +1,41 @@
+const { pages } = require('../pages/Pages');
+const users = require('../data/users');
+
+async function verifyNameSorting(order, option) {
+    const sortedNames = await pages.inventoryPage.sortedItemsNames(order);
+    await pages.inventoryPage.selectSortOption(option);
+    const expectedNamesOrder = await pages.inventoryPage.getNames(order);
+    expect(sortedNames).toEqual(expectedNamesOrder);
+}
+
+async function verifyPriceSorting(order, option) {
+    const sortedPrices = await pages.inventoryPage.sortedItemsPrices(order);
+    await pages.inventoryPage.selectSortOption(option);
+    const expectedPricesOrder = await pages.inventoryPage.getPrices(order);
```

Comment:
await pages.inventoryPage.getPrices(order); should we pass order here as a parameter?

Reply:
@AlinaZhykharieva correct, it's removed. thanks